test/functional/feature_fee_estimation.py

```
@@ -214,6 +218,153 @@ def test_feerate_mempoolminfee(self):
+
+        fee_dat = self.nodes[0].chain_path / "fee_estimates.dat"
+
+        # Stop the node and backdate the fee_estimates.dat file more than MAX_FILE_AGE
+        last_modified_time = time.time() - (MAX_FILE_AGE + 1) * SECONDS_PER_HOUR
+        os.utime(fee_dat, (last_modified_time, last_modified_time))
+
+        # Restart node with -acceptstalefeeestimates option to ensure fee_estimate.dat file is read
+        self.start_node(0,extra_args=["-acceptstalefeeestimates"])
+        assert_equal(self.nodes[0].estimatesmartfee(1)["feerate"], fee_rate)
+
+    def test_old_fee_estimate_file(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_old_fee_estimate_file`

Add return type annotation: `None`

(ANN201)